### PR TITLE
Fixed unresponsive terminal issue

### PR DESCRIPTION
**Pull request by IamRaviTejaG-zz**
Fixes issue #12

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -154,6 +154,8 @@ program
   .description('Start game in online mode')
   .option('-f, --friendly', 'Start playing online mode among friends (max 5)')
   .action((options) => {
+    if (options.friendly === undefined)
```
Comment (knrt10) [inline index.js]: I think `if(!options.friendly)` should work fine. Rest all looks good.

**Review by knrt10 — CHANGES_REQUESTED** [review]

**Comment by IamRaviTejaG-zz** [comment]
Made suggested changes. Review and merge.

**Review by knrt10 — APPROVED** [review]

**Comment by knrt10** [comment]
Looks good. Thanks @IamRaviTejaG

**Comment by IamRaviTejaG-zz** [comment]
@knrt10 Been editing my fork?! I'm confused.

**Comment by knrt10** [comment]
I dont't know what happend. Please close this PR.

**Pull request closed without merging**